specification/loadtestservice/LoadTestService/tspconfig.yaml

```diff
@@ -57,5 +60,5 @@ options:
     partial-update: true
     generate-tests: false
     generate-samples: false
-    service-name: Load Testing
+    service-name: Load Test
```

Comment:
@mjmadhu 
Would you let me know the reason why you change the `service-name`?

This value affect name of Java SDK class `LoadTestingServiceVersion`.
https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/loadtesting/azure-developer-loadtesting/src/main/java/com/azure/developer/loadtesting/LoadTestingServiceVersion.java
With this change, this class would be named `LoadTestServiceVersion`, and it is a breaking change.